lib/deep_merge/rails_compat.rb

```diff
@@ -13,12 +13,6 @@ def deeper_merge!(source, options = {})
       default_opts = {:preserve_unmergeables => false}
       DeepMerge::deep_merge!(source, self, default_opts.merge(options))
     end
-
-    # deeper_merge will merge and skip any unmergeables in destination hash
-    def deeper_merge(source, options = {})
-      default_opts = {:preserve_unmergeables => true}
-      DeepMerge::deep_merge!(source, self, default_opts.merge(options))
-    end
```

Comment:
I agree with the idea of this PR.  Although, we should probably deprecate this first in a release, then drop it after.

Reply:
please make that PR then, not using this library anymore